docs/visual-basic/misc/bc33013.md

```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: Operators cannot be declared '<keyword>"
```

Comment:
When the last word is surrounded by single-quotes, the closing single-quote is dropped in the description.

Reply:
fixed the ones I could find in this PR